common/components/card/template.njk

```diff
@@ -68,5 +69,15 @@
     {% if params.insetText %}
       {{ govukInsetText(params.insetText) }}
     {% endif %}
+
+    {% if params.isLockout %}
+      <div class="card-warning-text">
```

Comment:
```suggestion
      <div class="<div class="govuk-!-padding-top-3"></div>">
```

We can use the padding styles from the design system here: https://design-system.service.gov.uk/styles/spacing/